.mergify.yml

```diff
@@ -34,6 +46,7 @@ pull_request_rules:
       - author~=dependabot\[bot\]|dependabot-preview\[bot\]
       - "#approved-reviews-by>=1"
       - label!=submodules
+      - -title~=(?i)pyinstaller
```

Comment:
perhaps use `not:` here also?

Reply:
The mergify check doesn't seem to like using `not:` under another condition?

Reply:
That is really weird. I guess they don't specifically show how this is supposed to work, but it does say the depth is limited to 8...?

https://docs.mergify.com/conditions/#combining-conditions-with-operators

Reply:
Yeah.  It's strange.  Their example only has `not:` at the top.